packages/next/src/build/index.ts

```diff
@@ -1286,11 +1286,13 @@ export default async function build(
 
                 if (pageType === 'pages') {
                   pagePath =
-                    pagesPaths.find(
-                      (p) =>
+                    pagesPaths.find((p) => {
+                      p = p.replace(/[\\/]/, '/')
```

Comment:
```suggestion
                      p = p.replace(/[\\/]/g, '/')
```
Should we re-use https://github.com/vercel/next.js/blob/canary/packages/next/src/shared/lib/page-path/normalize-path-sep.ts